test/unit/spec/v2/view-builder/views/ChallengeWebauthnView_spec.js

```diff
@@ -223,4 +222,70 @@ describe('v2/view-builder/views/webauthn/ChallengeWebauthnView', function() {
       })
       .catch(done.fail);
   });
+
+  it('shows correct text when Can\'t verify? is clicked', function() {
+    jest.spyOn(webauthn, 'isNewApiAvailable').mockReturnValue(true);
+    jest.spyOn(BrowserFeatures, 'isSafari').mockReturnValue(false);
+    testContext.init();
+    expect(testContext.view.$('.idx-webauthn-verify-text').text()).toMatchInlineSnapshot(
+      '"You will be prompted to use a security key or biometric verification (Windows Hello, Touch ID, etc.). Follow the instructions to complete verification."'
+    );
+    expect(testContext.view.$('.js-cant-verify').length).toBe(1);
+    expect(testContext.view.$('.js-help-description').css('display')).toBe('none');
+    const cantVerifyLink = document.getElementsByClassName('link js-cant-verify');
+    cantVerifyLink[0].click();
+    expect(testContext.view.$('.js-help-description').css('display')).toBe('block');
+    expect(testContext.view.$('.js-help-description').first().html()).toMatchInlineSnapshot(
+      '"<h3>Are you trying to use a biometric authenticator?</h3><br><p>Biometric authenticators (fingerprint, face recognition, PIN) will only work on the same device on which they were set up.</p><br><p>If available, set up another security method on the device you used to set up your biometric authenticator.</p><br><h3>Are you trying to use a security key?</h3><br><p>If you have set up a security key, insert it in a USB port when prompted by the browser and tap on the button or gold disk. Security keys can work on multiple devices.</p><br>"'
+    );
+  });
+
+  it('shows additional text when Can\'t verify? is clicked from Okta_Authenticator', function() {
+    testContext.init = (
```

Comment:
instead of overriding `init` here, just make the changes to the `init` function at the beginning of this file. The only difference is that here you have a value for `app`, so you can handle that in the common init for all tests
```
const appStateOptions = {
  currentAuthenticator,
  authenticatorEnrollments,
};
if (app) {
  appStateOptions.app = app;
}
const appState = new AppState(appStateOptions);
```
The other changes to use jest spies and expects are good for all tests